README.md

```diff
@@ -1,12 +1,12 @@
 OpenAL-soft
 ===========
 
-CI status :  [![Build Status](https://travis-ci.org/kcat/openal-soft.svg?branch=master)](https://travis-ci.org/kcat/openal-soft)
+`master` branch CI status :  [![Build Status](https://travis-ci.org/kcat/openal-soft.svg?branch=master)](https://travis-ci.org/kcat/openal-soft) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/github/kcat/openal-soft?branch=master&svg=true)](https://ci.appveyor.com/api/projects/status/github/kcat/openal-soft?branch=master&svg=true)
 
 OpenAL Soft is an LGPL-licensed, cross-platform, software implementation of the OpenAL 3D audio API. It's forked from the open-sourced Windows version available originally from openal.org's SVN repository (now defunct).
 OpenAL provides capabilities for playing audio in a virtual 3D environment. Distance attenuation, doppler shift, and directional sound emitters are among the features handled by the API. More advanced effects, including air absorption, occlusion, and environmental reverb, are available through the EFX extension. It also facilitates streaming audio, multi-channel buffers, and audio capture.
 
-More information available on the [official website](http://openal-soft.org/)
+More information are available on the [official website](http://openal-soft.org/)
```

Comment:
I think "is" would be a better word choice there than "are". Also, a better header title would be OpenAL Soft, rather than OpenAL-soft.

Reply:
Oh, that's my stupid brain thinking in French while writing English again. I'm going to fix that, and remove the '-'